javascript/javascript1/week1/app.js

```diff
@@ -0,0 +1,73 @@
+//STEP 2
+//user freecodecamp fccdc38edef-ba87-461c-96cd-bf07dcfc8098
+
+//STEP 3
+
+/*Age-ify*/
+var yearOfBirth = 1987; 
+var yearFuture =  2027;
+var age = yearFuture - yearOfBirth;
+console.log("You will be " + age + " years old in " + yearFuture + ".");
+
+/*Goodboy-Oldboy */
+var dogYearOfBirth = 2020;
+var dogYearFuture = 2030;
+var dogYear = dogYearFuture - dogYearOfBirth;
+var shouldShowResultInDogYearsld = true;
+
+if(shouldShowResultInDogYearsld){
+    dogYear *= 7;
+    console.log("Your dog will be " + dogYear + " dog years old in " + dogYearFuture);
+} else{
+    console.log("Your dog will be " + dogYear + " human years old in " + dogYearFuture);
+}
+
+dogYearOfBirth = 2020;
+dogYearFuture = 2030;
+dogYear = dogYearFuture - dogYearOfBirth;
+shouldShowResultInDogYearsld = false;
+
+if(shouldShowResultInDogYearsld){
+    dogYear *= 7;
+    console.log("Your dog will be " + dogYear + " dog years old in " + dogYearFuture);
+} else{
+    console.log("Your dog will be " + dogYear + " human years old in " + dogYearFuture);
+}
+
+/*Housey Pricey*/
+
+
+var houseDepth = 10;
+var houseWidth = 8;
+var houseHeight = 10;
+var gardenSizeInM2 = 100;
+var volumeInMeters = houseDepth * houseWidth * houseHeight;
+var housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+var friendPayment = 2500000;
+
+if(housePrice - friendPayment < 0){
```

Comment:
You can use `>` : higher then operator 
```suggestion
if(housePrice > friendPayment < 0){
```

Reply:
Yes, I missed that option.